### ISSUE TITLE
Feature Request: add options on wp-config as constants

### ISSUE THREAD

**Issue by planetahuevo** [WPManageNinja/fluent-smtp#47]
Hi, 
First of all, thank you very much for the plugin and the development. This is amazing. 
I am still using because of this feature: https://wpmailsmtp.com/docs/how-to-secure-smtp-settings-by-using-constants/
which allow me to setup and deploy any website from the config file without any need to login into the WordPress for that.

It would be a great feature to have. I hope you can consider it. 

Thank you!

**Comment by petertwise** [comment]
+1

We are setting up MailTrap for our staging/testing sites and SES for production. It would be awesome to be able to put the full settings, including port, host, etc into wp-config.php

**Comment by planetahuevo** [comment]
Yeah, still cannot switch due to this issue. There are some codes, but not all of them. And half of them does not solve the problem. 
I hope they can add this in the future. I would love to migrate all my clients to this plugin.

**Comment by versusbassz** [comment]
+1  
The ability to configure a plugin via wp-config.php is a killer feature for me and the main reason why I don't migrate all my sites from `wp-mail-smtp` to this `fluentsmtp`.
